### DISCUSSION TITLE
Verthash mining, running hidden, unless other applications that listen to the value to run with a visible console

### DISCUSSION THREAD

**Discussion by mderoo75** [RainbowMiner/RainbowMiner#1675] [Q&A] [answered]
I notice, when running a benchmark, that the Verthash program is running hidden and not visible console/ GUI screen application (the only one, that's not respecting the value to run visible)

**Comment by RainbowMiner** [comment, marked as answer]
This is by design - that one miner doesn't have an API to read the current hashrate. To handle this, we need to grab that information from the miner's console output and parse it. When running on a Windows machine, this console grabbing is possible only, if we run the miner hidden, in the background.
